api.swagger.yaml

```diff
@@ -11175,27 +11583,207 @@ components:
                   format: uuid
                   example: '01234567-89ab-cdef-1234-56789abcdef0'
             team:
-              description: 'The team targeting this custom content.'
+              description: 'The team targeting this custom content.'
```

Comment:
*custom content typo

There are quite a few of these, could the descriptions be reviewed?

Reply:
I just realised this is to do with webforms anyway, which is not required for this task.

Reply:
Pretty sure I've got all the "custom content" typos out, they where all over at one point.

Reply:
Are custom content changes expected in this PR?